ibc_data/all_contrasts.tsv

```diff
@@ -673,50 +673,50 @@ LocalizerAbstraction	localizer_objects-other	objects	rest of categories	objects
-Abstraction	faces_geometry-faces_other	geometry render of human faces	edge and photo renders of human faces	geometry vs edge and photo render of faces	['visual_face_recognition','face_perception','visual_form_recognition']
-Abstraction	faces_edge-faces_other	edges render of faces	geometry and photo renders of faces	edge vs geometry and photo render of faces	['visual_face_recognition','face_perception','edge_detection']
-Abstraction	faces_photo-faces_other	photos of human faces	geometry and edge renders of human faces	photo vs geometry and edge render of faces	['visual_face_recognition','face_perception','visual_representation']
-Abstraction	flora_geometry-flora_other	geometry render of flora	edge and photo renders of flora	geometry vs edge and photo render of flora	['naturalistic_scenes','visual_shape_recognition','visual_form_recognition']
-Abstraction	flora_edge-flora_other	edges render of flora	geometry and photo renders of flora	edge vs geometry and photo render of flora	['naturalistic_scenes','visual_shape_recognition','edge_detection']
-Abstraction	flora_photo-flora_other	photos of flora	geometry and edge renders of flora	photo vs geometry and edge render of flora	['naturalistic_scenes','visual_shape_recognition','visual_representation']
-Abstraction	objects_geometry-objects_other	geometry render of objects	edge and photo renders of objects	geometry vs edge and photo render of objects	['visual_object_recognition','visual_form_recognition']
-Abstraction	objects_edge-objects_other	edges render of objects	geometry and photo renders of objects	edge vs geometry and photo render of objects	['visual_object_recognition','edge_detection']
-Abstraction	objects_photo-objects_other	photos of objects	geometry and edge renders of objects	photo vs geometry and edge render of objects	['visual_object_recognition','visual_representation']
-Abstraction	places_geometry-places_other	geometry render of places	edge and photo renders of places	geometry vs edge and photo render of places	['visual_place_recognition','visual_form_recognition']
-Abstraction	places_edge-places_other	edges render of places	geometry and photo renders of places	edge vs geometry and photo render of places	['visual_place_recognition','edge_detection']
-Abstraction	places_photo-places_other	photos of places	geometry and edge renders of places	photo vs geometry and edge render of places	['visual_place_recognition','visual_representation']
+Abstraction	edge-other	edge renders of all categories	geometry and photo renders of all categories	edge vs geometry and photo render	['visual_representation']
+Abstraction	photo-other	photos of all categories	geometry and edge renders of all categories	photo vs geometry and edge render	['visual_place_recognition','visual_object_recognition','naturalistic_scenes','visual_body_recognition','visual_face_recognition','animacy_perception']

```

Comment:
Just curious: `animacy perception` is not for things that move, that are active in some way ?

Reply:
The cognitive atlas includes the concept of biological motion which makes it slightly unsuitable for static stimuli.
I'd rather not ahve it here.